jlm/llvm/frontend/InterProceduralGraphConversion.cpp

```diff
@@ -541,10 +541,6 @@ ConvertThreeAddressCode(
   {
     Convert<CallNode, CallOperation>(threeAddressCode, region, variableMap);
   }
-  else if (is<LoadVolatileOperation>(&threeAddressCode))
```

Comment:
I don't quite understand why this is removed entirely, do we not need to convert the operation?

Reply:
Look at the else case. A normal SimpleNode with the LoadVolatileOperation is created instead of a dedicated node.